### ISSUE TITLE
Thank You Note

### ISSUE THREAD

**Issue by Zacharias030** [eczarny/spectacle#945]
Dear Eric Czarny et al.,

I just noticed that the beloved spectacle has been discontinued. Spectacle is such a core part of my workflow that I never expected it to not be there one day.

Just wanted to say thanks for the great work before I leave!

Best,
Zach

**Comment by Zacharias030** [comment]
(please feel free to close this issue at your convenience)

**Comment by broven** [comment]
Thanks a Ton

**Comment by jfo** [comment]
I've been using this app since 2014 and I'd like to say that it has been a real pleasure. I've always configured it on new setup and then just forgotten about it, it has Just Worked, and that's a really high bar.

Thank you for your work on this, it is appreciated!

**Comment by jamesfawcett** [comment]
I'd also like to say thanks for this fantastic plugin! I just set up a new work mac today, it's the first thing I always install! I really hope it continues working into the future 👍

**Comment by jimmy927** [comment]
I am just in the process of switching from Mac OS to Microsoft Windows.

I have ported over all my python-code to windows which is littered with UNIX-specific stuff ...
I made all my docker & docker-compose work nativly!
I am getting used to the Windows Terminal, which is actually pretty decent.

My biggest problem right now: No spectacles - i think that says a lot!

Is there something like this for windows ?!?

**Comment by ArtSabintsev** [comment]
Oh wow - discontinued? That makes me sad! I've been using this product for so many years. I love it - thank you!

What is everyone using as an alternative?

**Comment by ryancwalsh** [comment]
Spectacle is truly amazing! Thank you!

**Comment by zachberger** [comment]
Adding my thanks!

**Comment by masonkoh** [comment]
*salute*

**Comment by Kirill89** [comment]
Spectacle was my "must to install" tool for many years!
Thank you!

**Comment by monfortm** [comment]
Yeah, btw thanks! The software is always on on my machine since in installed it a few years ago.

**Comment by scsven** [comment]
@eczarny  I have been using Spectacle since 2017 and it's pretty cool. Thanks for your work !

**Comment by jamesfawcett** [comment]
Can anyone confirm does Spectacle work in Big Sur? I don't want to update unless everything works, I'm a big Catalina fan!

**Comment by ricardosllm** [comment]
> Can anyone confirm does Spectacle work in Big Sur? I don't want to update unless everything works, I'm a big Catalina fan!

@jamesfawcett works like a charm, even on an M1 MBP (I `brew install spectacle` btw)

I've actually tried Rectangle first but could not get used to it, I'll stay with spectacle while it works

Great work @eczarny and thank you sir 🙏

**Comment by 1oglop1** [comment]
@jimmy927 The think about windows is that UI is not stuck in 2001, 
you can use `WIN+<arrow>` to do what specacle did.

**Comment by zachberger** [comment]
There is also a spectacle mode available during setup. I haven't noticed
any difference after setting this.

**Comment by startswithaj** [comment]
Yeah huge thanks @eczarny - really loved this app. As others say set and forget. @zachberger thanks for opening this thread!

**Comment by nonoesp** [comment]
Thanks! ❤️

**Comment by prasskaly** [comment]
Thank you @eczarny ! Spectacle has been a timesaver and a great app !

**Comment by limianwang** [comment]
I cannot agree more. Thank you @eczarny for creating this project. Despite it not being maintained, it is still a core component that functions well for me on mac!

**Comment by sommerper** [comment]
I’ll continue using it until it no longer works. Thanks from me as well.

**Comment by eriksss** [comment]
still using it...

**Comment by joehanna** [comment]
Thank you! It is in my essential daily toolkit too
